src/app/content/constants.ts

```diff
@@ -18,3 +19,11 @@ export const highlightStyles: Array<{label: HighlightColorEnum, passive: string,
   {label: HighlightColorEnum.Purple, passive: '#cbcfff', focused: '#545ec8'},
   {label: HighlightColorEnum.Pink, passive: '#ffc5e1', focused: '#de017e'},
 ];
+
+export const getBaseColors = () => {
+  const bases: { [key: string]: string } = {};
+  const primaryColors: {[key: string]: {[key: string]: string}} = theme.color.primary;
+
+  Object.keys(primaryColors).forEach((color) => {bases[color] = primaryColors[color].base; });
```

Comment:
is all this getting the base property of the primary colors? maybe:
```
Object.fromEntries(Object.entries(theme.color.primary).map(([key, {base}]) => [key, base]))
```

or you could use the `mapValues` from lodash that works on objects

Reply:
I'm running into issues with both of those options. As I mentioned, `Object.fromEntries` is from a newer ES version than the one we're targeting, and `mapValues` returns an empty object always, including when I use an example from the lodash docs:
```
const users = {
  fred: { user: 'fred', age: 40 },
  pebbles: { user: 'pebbles', age: 1 },
};

console.log(mapValues(users, (o: any) => o.age));
// => {}

console.log(mapValues(users, 'age'));
  // => {0: false, 1: false, 2: false}
```